core/src/test/scala/org/apache/spark/deploy/history/HistoryServerSuite.scala

```diff
@@ -390,6 +390,8 @@ abstract class HistoryServerSuite extends SparkFunSuite with BeforeAndAfter with
     // a new conf is used with the background thread set and running at its fastest
     // allowed refresh rate (1Hz)
     stop()
+    Utils.deleteRecursively(storeDir)
+    assert(storeDir.mkdir())
```

Comment:
Looks okay. Maybe add a comment on top of it explaining why it is needed.

Reply:
Sure. Thank you!